src/twisted/words/protocols/irc.py

```diff
@@ -1238,6 +1248,9 @@ class IRCClient(basic.LineReceiver):
     _heartbeat = None
     heartbeatInterval = 120
 
+    decodeCodecs: list[str] = ["utf-8"]
```

Comment:
Oh, and we still have to use `List[str]` because we support old Python yet.

Reply:
I forgot that this is a quite new feature.